ingress-operator/pkg/controller/core.go

```diff
@@ -253,7 +253,6 @@ func MakeAnnotations(fni *faasv1.InferenceIngress, host string) map[string]strin
 	specJSON, _ := json.Marshal(fni)
 	annotations := make(map[string]string)
 
-	annotations["kubernetes.io/ingress.class"] = class
```

Comment:
Will this affect the ingress?

Reply:
Ingress does not need this annotation when setting the ingress type.

Reply:
```
 invalid: annotations.kubernetes.io/ingress.class: Invalid value: \"nginx\": can not be set when the class field is also set" 
```

Reply:
https://serverfault.com/questions/1061530/new-kubernetes-ingressclass-resource-and-only-specifying-ingressclassname-nginx